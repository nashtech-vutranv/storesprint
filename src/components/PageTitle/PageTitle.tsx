import React, {FC} from 'react'
import {Col} from 'react-bootstrap'

const PageTitle: FC<{title: string}> = ({title}) => {
  return (
    <Col>
      <div className='page-title-box'>
        <h4 className='page-title'>{title}</h4>
      </div>
    </Col>
  )
}

export default PageTitle
