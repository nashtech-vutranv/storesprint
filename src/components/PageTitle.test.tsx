import {render, waitFor} from '@testing-library/react'
import '@testing-library/jest-dom'
import {MemoryRouter} from 'react-router-dom'
import PageTitle from './PageTitle'

test('renders PageTitle', async () => {
  const tree = render(
    <MemoryRouter>
      <PageTitle
        breadCrumbItems={[{label: 'test-label', path: 'test-path'}]}
        title='page-title'
      />
    </MemoryRouter>
  )

  await waitFor(() => {
    expect(tree).toMatchSnapshot()
  })
})
