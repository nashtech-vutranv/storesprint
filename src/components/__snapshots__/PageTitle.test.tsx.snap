// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders PageTitle 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="row"
      >
        <div
          class="col"
        >
          <div
            class="page-title-box"
          >
            <div
              class="page-title-right"
            >
              <nav
                aria-label="breadcrumb"
              >
                <ol
                  class="breadcrumb m-0"
                >
                  <li
                    class="breadcrumb-item"
                  >
                    <a
                      href="/"
                    >
                      Hyper
                    </a>
                  </li>
                  <li
                    class="breadcrumb-item"
                  >
                    <a
                      href="test-path"
                    >
                      test-label
                    </a>
                  </li>
                </ol>
              </nav>
            </div>
            <h4
              class="page-title"
            >
              page-title
            </h4>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="row"
    >
      <div
        class="col"
      >
        <div
          class="page-title-box"
        >
          <div
            class="page-title-right"
          >
            <nav
              aria-label="breadcrumb"
            >
              <ol
                class="breadcrumb m-0"
              >
                <li
                  class="breadcrumb-item"
                >
                  <a
                    href="/"
                  >
                    Hyper
                  </a>
                </li>
                <li
                  class="breadcrumb-item"
                >
                  <a
                    href="test-path"
                  >
                    test-label
                  </a>
                </li>
              </ol>
            </nav>
          </div>
          <h4
            class="page-title"
          >
            page-title
          </h4>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
